src/index.html

```diff
@@ -26,7 +26,7 @@
       </ul>
       <ul class="toDoList"></ul>
       <article class="button-section">
-        <a href="#">Clear all completed</a>
+        <a href="#" id="clearCompleteTodo">Clear all completed</a>
       </article>
```

Comment:
- [ **OPTIONAL** ] I know the `a` tag is good to be used here. But the best one on the `Clear all completed` button will be a `<button>` tag, so I suggest you use it instead. But Good :+1: :+1:  job and happy coding :tada: :tada:

Reply:
Thank you. I will